src/main/blackbox/blackbox.c

```diff
@@ -1657,7 +1657,8 @@ void handleBlackbox(timeUs_t currentTimeUs)
 
 static bool canUseBlackboxWithCurrentConfiguration(void)
 {
-    return feature(FEATURE_BLACKBOX);
+    return feature(FEATURE_BLACKBOX) && 
+        (blackboxConfig()->device != BLACKBOX_SDCARD || feature(FEATURE_SDCARD));
```

Comment:
Certainly feasible. Currently not much checking like this is done in configurator, since a lot of it is hardware dependent, and there is just about no feedback about hardware in MSP. (i.e. if they select SDCard on a target that does not have SDCard, it should not allow them to choose SDCard logging...)